### DISCUSSION TITLE
Translation theme via Market

### DISCUSSION THREAD

**Discussion by tonybastienricher** [EvgeniyMukhamedjanov/liquid-ajax-cart#23] [Q&A] [answered]
Hello to all 👋,

We encounter a problem when the route.url links change. 

Some of our customers translate their sites with Shopify market and when the links change, adding to the cart in ajax doesn't work anymore.

For example :
https://pimentio.myshopify.com/
https://pimentio.myshopify.com/en

When I add a product via the English site, I get this error :

`Uncaught TypeError: can't access property "setActiveElement", this.cartNotification is null`

The shopping cart does not work anymore.

Have you already encountered this problem?

Thank you in advance for your help.

Best regards

**Comment by EvgeniyMukhamedjanov** [comment, marked as answer]
Hey @tonybastienricher .

I didn't check why it is happening but I guess it is because Shopify released new url cart routes that should be used if Shopify Markets is turned on.

I have one Shopify Markets shop in development right now and when I reach Ajax Cart part I'll definitely fix Liquid Ajax Cart as well to support all the URL. Hope it will be during 3—4 weeks.

If you create an issue for this in GitHub it would be helpful so you'll also be notified when I resolve this.

**Reply by tonybastienricher** [reply]
Hey @EvgeniyMukhamedjanov,
Thanks for your answer!
I will create an issue, I can help if needed.

**Reply by EvgeniyMukhamedjanov** [reply]
@tonybastienricher 
Thank you for the offer to help!
If you can find the reason why it doesn't work or even make a PR it would be great!
